akka-docs/src/main/paradox/typed/persistence-durable-state.md

```diff
@@ -0,0 +1,395 @@
+---
+project.description: Durable State with Akka Persistence enables actors to persist its state for recovery on failure or when migrated within a cluster.
+---
+# Durable State
+
+## Module info
+
+To use Akka Persistence, add the module to your project:
+
+@@dependency[sbt,Maven,Gradle] {
+  bomGroup=com.typesafe.akka bomArtifact=akka-bom_$scala.binary.version$ bomVersionSymbols=AkkaVersion
+  symbol1=AkkaVersion
+  value1="$akka.version$"
+  group=com.typesafe.akka
+  artifact=akka-persistence-typed_$scala.binary.version$
+  version=AkkaVersion
+  group2=com.typesafe.akka
+  artifact2=akka-persistence-testkit_$scala.binary.version$
+  version2=AkkaVersion
+  scope2=test
+}
+
+You also have to select durable state store plugin, see @ref:[Persistence Plugins](../persistence-plugins.md).
+
+
+@@project-info{ projectId="akka-persistence-typed" }
+
+## Introduction
+
+This model of Akka Persistence enables a stateful actor / entity to store the full state after processing each command instead of using event sourcing. This reduces the conceptual complexity and can be a handy tool for simple use cases. Very much like a CRUD based operation, the API is conceptually simple - a function from current state and incoming command to the next state which replaces the current state in the database. 
+
+```
+(State, Command) => State
+```
+
+The current state is always stored in the database. Since only the latest state is stored, we don't have access to any of the history of changes, unlike event sourced storage. Akka Persistence would read that state and store it in memory. After processing of the command is finished, the new state will be stored in the database. The processing of the next command will not start until the state has been successfully stored in the database.
+
+Akka Persistence also supports @ref:[Event Sourcing](persistence.md) based implementation, where only the _events_ that are persisted by the actor are stored, but not the actual state of the actor. By storing all events, using this model, 
+a stateful actor can be recovered by replaying the stored events to the actor, which allows it to rebuild its state.
+
+The database specific implementations can be added to existing Akka Persistence plugin implementations, starting with the JDBC plugin. The plugin would serialize the state and store as a blob with the persistenceId as the primary key. 
+
```

Comment:
I think this is still a bit confusing for someone who is used to a traditional '3-tiered' architecture. Should we add some text about how this approach is different? (i.e. since each entity lives on one node, consistency is guaranteed and reads can be served directly from memory - linking to the 'sharding' section below to find out how to easily make sure each entity lives on one node even in clustered applications)

Reply:
done ..